src/main/java/bob/Task.java

```diff
@@ -0,0 +1,57 @@
+package bob;
+
+import java.util.UUID;
+
+/**
+ * Base class representing a user TODO task.
+ */
+public class Task {
+
+    protected String description;
+    protected boolean done = false;
```

Comment:
According to our coding standard, Boolean variables/methods should be named to sound like booleans.
Maybe `isDone` would be better?